package.json

```diff
@@ -25,7 +25,7 @@
 		"test": "pnpm exec turbo run turbo:test",
 		"clean": "pnpm store prune && git clean -fx **/node_modules && pnpm i",
 		"preinstall": "npx only-allow pnpm",
-		"postinstall": "pnpm git:update-hooks && pnpm run --filter='./tools/monorepo-utils' build",
+		"postinstall": "pnpm git:update-hooks && pnpm run --filter='./tools/monorepo-utils' build && chmod +x tools/monorepo-utils/dist/index.js",
```

Comment:
This won't be required. This PR addresses permissions https://github.com/woocommerce/woocommerce/pull/37670